runtime/runtime_events.c

```diff
@@ -418,15 +418,15 @@ static void stw_create_runtime_events(
   caml_global_barrier();
 }
 
-CAMLprim value caml_runtime_events_start(void) {
+CAMLprim value caml_runtime_events_start(value vunit) {
```

Comment:
These three form part of the public API for runtime events, which is breaking one of the tests. Perhaps rename them to `caml_ml_runtime_events_start` (a la lots of the I/O primitives) and then either `#define caml_runtime_events_start` or an inline'd definition of `caml_runtime_events_start`?

Reply:
You're right, it's a good idea to separate the C API functions `caml_runtime_events_start` and the primitive `caml_ml_runtime_events_start`.  I went one step further and made the C functions `void -> void` instead of `void -> value`, as `void -> void` is the natural C type for these functions.  Let me know what you think about that.

Reply:
I was fairly sure I wouldn't need to suggest the change to change the return type to `void`, indeed 🙂

Reply:
I'm glad you thought of changing the return type to `void`!